nextflow.config

```diff
@@ -74,7 +74,7 @@ params {
     kallisto_quant_fraglen_sd  = 200
     save_merged_fastq          = false
     save_unaligned             = false
-    save_align_intermeds       = false
+    save_align_intermeds       = true
```

Comment:
Is there a need for this as default?

Reply:
Yeah, I think the alignment files are more of a necessary output for riboseq, while in rnaseq they're only a stop on a path to quantification. 

I might reverse it later, but for now it was intentional.